dso/model.py

```diff
@@ -712,6 +718,8 @@ class CustomRescoring:
     )
     matching_rules: list[str] = dataclasses.field(default_factory=list)
     comment: str | None = None
+    allowed_processing_time: str | None = None
+    due_date: datetime.date | None = None
```

Comment:
should we add a (maybe in post-init) validation that will raise if either both or none are set (i.e. XOR)